master/internal/api_model.go

```diff
@@ -157,8 +157,9 @@ func (a *apiServer) PatchModel(
 	currLabels := strings.Join(currModel.Labels, ",")
 	if req.Model.Labels != nil {
 		var reqLabelList []string
-		for i := 0; i < len(req.Model.Labels); i++ {
-			reqLabelList = append(reqLabelList, req.Model.Labels[i].Value)
+		for _, el := range req.Model.Labels.Values {
+			sval := el.String()
+			reqLabelList = append(reqLabelList, sval[14:len(sval)-1])
```

Comment:
this should be a bit more reliable and readable:

```suggestion
			reqLabelList = append(reqLabelList, el.GetStringValue())
```

also we could check if `el.GetKind()` is stringish for extra reliability